definitions/marts/looker_studio_marts/support_tickets/ls_support_ticket_volumes.sqlx

```diff
@@ -0,0 +1,238 @@
+    updated_at,
+    status_updated_at,
+    LOWER(stm.subject) AS subject
+  FROM
+    `ecf-bq.zendesk_support_ticket_data.support_ticket_details_anonymised` AS sta
+  LEFT JOIN
+    `ecf-bq.static_tables.support_ticket_details_migrated` AS stm
+  ON
+    sta.ticket_id = stm.destination_id
+  WHERE
+    group_name LIKE 'CPD%' ),
+  duplicate_tickets AS (
+  SELECT
+    DISTINCT ticket_id,
+    CASE WHEN'closed_by_merge' IN UNNEST(tags) AND status = 'closed' THEN TRUE
+      ELSE FALSE
+  END
+    AS is_duplicate,
+  FROM
+    cpd_prog_data ),
+  -- First classification layer: uses the group_name to assign classification
+  first_classification_layer AS (
+  SELECT
+    cpd_prog_data.*,
+    CASE
+    -- WHEN'closed_by_merge' IN UNNEST(tags) then 'Duplicate ticket'
+      WHEN group_name IN ('CPD - ECTE', 'CPD - 2nd Line- LP and DP Queries', 'CPD - 2nd Line: Appropriate Bodies & Regs', 'CPD - 2nd line - Withdrawing or reinstating ECTs/mentors', 'CPD - 3rd Line Support (Dfe admin)', 'CPD - ECF', 'CPD - ECF - Access to DfE service', 'CPD - ECF - ECT registration issues', 'CPD - ECF - Mentor registration issues', 'CPD - ECF - Partnerships', 'CPD - 2nd line ECF review (policy)', 'CPD - 2nd line- Unable to change cohort due to declaration errors on the service') THEN 'CPD - ECTE'
+      WHEN group_name IN ('CPD - ECF - 2nd line Funding Enquiries',
+      'CPD - ECF Funding',
+      'CPD - ECTE Funding') THEN 'ECTE - Funding'
+      WHEN group_name = 'CPD - Get a TRN' THEN 'NPQ - Get a TRN'
+      WHEN group_name IN ('CPD - Induction and AB',
+      'CPD - Legacy Induction') THEN 'ECTE - Induction and AB'
+      WHEN group_name IN ('CPD - 2nd Line - NPQ Digital', 'CPD - 2nd Line NPQ Policy Provider Engagement', 'CPD - 2nd line - NQTplus1 overpayments', 'CPD - 3rd Line - NPQ Digital - Developer to action', 'CPD - NPQ', 'CPD - NPQ - General Enquiries', 'CPD - 2nd Line-NPQ TSF') THEN 'CPD - NPQ'
+      WHEN group_name = 'CPD - 2nd line - NPQ Funding' THEN 'NPQ - Funding'
+      WHEN group_name IN ('CPD - Archive','CPD - 2nd line - Partnership Amendments','CPD - 2nd line: Schools and Teachers Team') THEN NULL
+      ELSE group_name
+  END
+    AS classification,
+    false as proportionate_assignment
+  FROM
+    cpd_prog_data ),
+  -- Second classification layer: classifies support tickets based on tags and issue_types associated to each ticket
+  second_classification_layer AS (
+  SELECT
+    fcl.* EXCEPT(classification, proportionate_assignment),
+    CASE
+      WHEN EXISTS ( SELECT 1 FROM UNNEST(issue_type || tags) AS val WHERE LOWER(val) LIKE '%induction_and_ab%' ) THEN 'ECTE - Induction and AB'
+      WHEN EXISTS (
+    SELECT
+      1
+    FROM
+      UNNEST(issue_type || tags) AS val
+    WHERE
+      LOWER(val) LIKE '%funding%'
+      AND LOWER(val) NOT LIKE '%npq%' ) THEN 'ECTE - Funding'
+      WHEN EXISTS ( SELECT 1 FROM UNNEST(issue_type || tags) AS val WHERE LOWER(val) LIKE '%funding%' AND LOWER(val) LIKE '%npq%' ) THEN 'NPQ - Funding'

```

Comment:
Where you have multiple LIKE conditions you can use LIKE ANY and similar depending on the exact conditions. See https://cloud.google.com/bigquery/docs/reference/standard-sql/operators#like_operator_quantified